qutebrowser/completion/models/miscmodels.py

```diff
@@ -117,9 +119,12 @@ def delete_buffer(data):
         if tabbed_browser.shutting_down:
             continue
         tabs = []
-        for idx in range(tabbed_browser.widget.count()):
+        num_tabs = tabbed_browser.widget.count()
+        idx_width = int(log10(num_tabs)+1) if num_tabs else 0
```

Comment:
nit: I don't think `if num_tabs else 0` is necessary, since the following loop won't run if `num_tabs == 0`

Reply:
This is needed because `log10(0)` will throw an error, but I agree it's a little weird...

Reply:
Could just do `len(str(num_tabs))` instead?